mne_bids/tests/test_write.py

```diff
@@ -196,6 +197,15 @@ def test_fif(_bids_validate):
         assert 'part' not in FILE
     assert ii < 1
 
+    # test keyword mne-bids anonymize
+    raw = mne.io.read_raw_fif(raw_fname)
+    output_path = _TempDir()
+
+    bids_output_path = write_raw_bids(raw, bids_basename, output_path,
+                                      events_data=events_fname,
+                                      event_id=event_id, anonymize=True,
+                                      overwrite=False)
```

Comment:
can you add the `anonymize=True` argument to all the file types in the tests to make sure they all work